resources/js/Shared/Layout/AdminNavigation.vue

```diff
@@ -1,65 +1,84 @@
+            <Link href="/">
+              <img class="h-10 sm:hidden" src="@/assets/scooter.png" alt="escooter logo">
+            </Link>
+            <button type="button" class="-m-2.5 rounded-md px-2.5 text-gray-700 sm:pt-4"
+                    @click="toggleMobileMenu"
+            >
+              <span class="sr-only">Close menu</span>
+              <XMarkIcon class="h-6 w-6" aria-hidden="true" />
+            </button>
+          </div>
+          <div class="mt-6 flow-root">
+            <div class="-my-6 divide-y divide-gray-500/10">
+              <div class="space-y-2 py-6">
+                <Link v-for="item in navigation" :key="item.name"
+                      :class="{'bg-blumilk-50': $page.url.startsWith(item.href)}" :href="item.href"
+                      class="-mx-3 block rounded-lg px-3 py-2 text-base font-semibold leading-7 text-gray-800 hover:bg-blumilk-25"
+                >
+                  {{ item.name }}
+                </Link>
+              </div>
+            </div>
+          </div>
+        </DialogPanel>
+      </Dialog>
 
       <ul class="hidden h-full items-center text-sm font-medium text-gray-800 sm:flex md:mt-12 md:flex-col md:items-stretch md:space-y-2">
-        <Link class="flex h-full md:h-fit" href="/admin/dashboard">
-          <li
-            :class="props.url === '/admin/dashboard' ? 'bg-blumilk-50' : ''"
-            class="mx-auto flex w-11/12 items-center bg-blumilk-25 px-6 hover:bg-blumilk-50 md:rounded-lg md:px-2 md:py-3"
-          >
-            <ClipboardIcon class="h-7 w-7" />
-            <span class="ml-3 hidden md:flex">Dashboard</span>
-          </li>
-        </Link>
-
-        <Link class="flex h-full md:h-fit" href="/admin/dashboard/countries">
-          <li :class="$page.url.startsWith('/admin/dashboard/countries')? 'bg-blumilk-50' : ''"
-              class="mx-auto flex w-11/12 items-center bg-blumilk-25 px-6 hover:bg-blumilk-50 md:rounded-lg md:px-2 md:py-3"
-          >
-            <FlagIcon class="h-7 w-7" />
-            <span class="ml-3 hidden md:flex">Countries</span>
-          </li>
-        </Link>
-        <Link class="flex h-full md:h-fit" href="/admin/dashboard/cities">
-          <li :class="$page.url.startsWith('/admin/dashboard/cities') ? 'bg-blumilk-50' : ''"
-              class="mx-auto flex w-11/12 items-center bg-blumilk-25 px-6 hover:bg-blumilk-50 md:rounded-lg md:px-2 md:py-3"
-          >
-            <MapPinIcon class="h-7 w-7" />
-            <span class="ml-3 hidden md:flex">Cities</span>
-          </li>
-        </Link>
-
-        <Link class="flex h-full md:h-fit" href="#">
-          <li
-            :class="props.url === '/admin/dashboard/statistics' ? 'bg-blumilk-50' : ''"
-            class="mx-auto flex w-11/12 items-center bg-blumilk-25 px-6 hover:bg-blumilk-50 md:rounded-lg md:px-2 md:py-3"
+        <Link v-for="item in navigation" :key="item.name" :href="item.href" class="flex h-full md:h-fit">
```

Comment:
I would recommend changing all `<Link/>` components to `<InertiaLink/>` as defined in app.js.